ci/redmine_bugzilla.py

```diff
@@ -0,0 +1,153 @@
+"""
+This module does a few things:
+
+1) Verify that all Bugzilla bugs which point to a Redmine issues also have
+Redmine issues pointing back to Bugzilla bugs. E-mail a list of people if
+only a uni-directional relationship exists so they can be fixed manually. The
+EMAIL_RECEIVERS constant specifies the recipient list.
+
+2) For all Redmine issues that track Bugzilla bugs, ensure that the
+Bugzilla external tracker description, status, and priority all mirror the
+Redmine issue state.
+
+Requirements:
+The Redmine and Bugzilla credentials are expected from to be read from a
+file in ini format. See the example below. The location of this file is set
+by the REDMINE_BUGZILLA_CONF environment variable.
+
+<SNIP>
+[bugzilla]
+username = XXXX
+password = XXXX
+
+[redmine]
+key = XXXX
+</SNIP>
+"""
+
+
+import ConfigParser
+import os
+
+from bugzilla.rhbugzilla import RHBugzilla
+from redmine import Redmine
+import urllib3.contrib.pyopenssl
+
+# Here due to InsecurePlatformWarning
+# https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning
+urllib3.contrib.pyopenssl.inject_into_urllib3()
+
+
+BUGZILLA_URL = 'https://bugzilla.redhat.com'
+REDMINE_URL = 'https://pulp.plan.io'
+
+
+EMAIL_RECEIVERS = ['bmbouter@gmail.com']
+FROM_ADDRESS = 'bbouters@redhat.com'
+EMAIL_SUBJECT = 'Pulp upstream/downstream Bug Report'
```

Comment:
This seems a little different than what was in the Yaml file. Does the YAML file override these?